### ISSUE TITLE
When the JSON output to top hour one unnecessary comma is inserted.

### ISSUE THREAD

**Issue by kometchtech** [vergoh/vnstat#17]
And insert the JSON output in one unnecessary commas, therefore in jq command cannot parse error occurs.

This is in recognition that the bug right?

```
# vnstat --json
{"vnstatversion":"1.13","jsonversion":"1","interfaces":[{"id":"eth0","nick":"eth0","created":{"date":{"year":2015,"month":4,"day":6}},"updated":{"date":{"year":2015,"month":4,"day":6},"time":{"hour":16,"minutes":2}},"traffic":{"total":{"rx":33464,"tx":920},"days":[{"id":0,"date":{"year":2015,"month":4,"day":6},"rx":33464,"tx":920}],"months":[{"id":0,"date":{"year":2015,"month":4},"rx":33464,"tx":920}],"tops":[],"hours":[,{"id":15,"date":{"year":2015,"month":4,"day":6},"rx":33367,"tx":894},{"id":16,"date":{"year":2015,"month":4,"day":6},"rx":97,"tx":26}]}}]}
[root@localhost ~]# vnstat --json | jq .
parse error: Expected value before ',' at line 1, column 425
```

**Comment by vergoh** [comment]
I think this is the same bug as reported and fixed in issue #13. Could you verify using the current master branch version if this can still be replicated?

**Comment by kometchtech** [comment]
Sorry to say this.
vnstat 1.13 was confirmed the behavior in the package.
confirmed to work fine once you get from git.

Sorry, not yet confirmed.
